torchao/prototype/low_bit_optim/subclass_fp8.py

```diff
@@ -47,7 +91,9 @@ def __init__(self, codes: Tensor, scale: Tensor):
         assert scale.ndim == 1
         self.codes = codes
         self.scale = scale
+        self.k = k
         self.block_size = codes.numel() // scale.numel()
+        self.sqrt_minmax_exp = sqrt_minmax_exp
 
     def __tensor_flatten__(self):
         return self.tensor_attrs, []
```

Comment:
When `k` and `sqrt_minmax_exp` is not None, you need to return them here also.

Reply:
should i pass them instead of empty array?

Reply:
The first returned value (currently `self.tensor_attrs`) is a list of strings containing the names of tensor attributes. In this case, when there is no dynamic range extension, it's just `"codes", "scale"`. However, when there is dynamic range extension, you need to also add `"k", "sqrt_minmax_exp"`. IIRC, when they are None, you are not supposed to include them.

Reply:
Thank you for  explaining it. i have added them. looking forward to your approval